lib/govuk_app_config/govuk_error.rb

```diff
@@ -10,4 +10,10 @@ def self.notify(exception_or_message, args = {})
 
     Raven.capture_exception(exception_or_message, args)
   end
+
+  def self.configure(&block)
```

Comment:
Do you actually need `&block` here given that you're using `yield`?

Reply:
We do not, thanks!